src/server/mods.cpp

```diff
@@ -66,8 +67,18 @@ void ServerModManager::loadMods(ServerScripting *script)
 		}
 		std::string script_path = mod.path + DIR_DELIM + "init.lua";
 		infostream << "  [" << padStringRight(mod.name, 12) << "] [\""
-			   << script_path << "\"]" << std::endl;
+			<< script_path << "\"]" << std::endl;
+		u64 t = porting::getTimeUs();
```

Comment:
just use c++11 std::chrono, it's for this

Reply:
Example here
http://www.techiedelight.com/measure-elapsed-time-program-chrono-library/